File: CSharpREST/Chart/ConvertChartToImage.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using Aspose.Cells.Cloud.Examples;

namespace Aspose.Cells.Cloud.Examples.Chart
{
    class ConvertChartToImage
    {
        static void Main()
        {
            string inputFile = Common.GetPath(System.Reflection.MethodBase.GetCurrentMethod().DeclaringType, "Sample.xlsx");
            string outputFile = Common.GetPath(System.Reflection.MethodBase.GetCurrentMethod().DeclaringType, "Chart.png");

            Common.ExecuteRequest(
                Common.SignRequestUrl(Common.BASE_PATH + "storage/file/Sample.xlsx"),
                "PUT",
                System.IO.File.ReadAllBytes(inputFile)
                );

            byte[] buf = Common.ExecuteRequest(
                Common.SignRequestUrl(Common.BASE_PATH + "cells/Sample.xlsx/worksheets/Sheet1/charts/0?format=png"),
                "GET"
                );

            System.IO.File.WriteAllBytes(outputFile, buf);

            Common.Pause();
        }
    }
}

File: VisualBasicSDK/Chart/ConvertChartToImage.vb
Namespace Aspose.Cells.Cloud.Examples
    Public Class ConvertChartToImage
        Shared Sub Main()
            Dim inputFile As String = Common.GetPath(System.Reflection.MethodBase.GetCurrentMethod().DeclaringType, "Sample.xlsx")
            Dim outputFile As String = Common.GetPath(System.Reflection.MethodBase.GetCurrentMethod().DeclaringType, "Chart.png")

            Common.GetStorageSdk().File.UploadFile(
                inputFile,
                System.IO.Path.GetFileName(inputFile)
                )

            Common.GetCellsSdk().Charts.GetChartInSomeFormat(
                System.IO.Path.GetFileName(inputFile),
                "Sheet1",
                0,
                Global.Aspose.Cloud.CellsChartOutputFormat.Png,
                "",
                System.IO.Path.GetFileName(outputFile),
                )

            Common.GetStorageSdk().File.DownloadFile(
                System.IO.Path.GetFileName(outputFile),
                outputFile
                )

        End Sub
    End Class
End Namespace
